opencensus/stats/measure_to_view_map.py

```diff
@@ -145,3 +142,11 @@ def get_metrics(self, timestamp):
                 metric = metric_utils.view_data_to_metric(vd, timestamp)
                 if metric is not None:
                     yield metric
+
+    # TODO(issue #470): remove this method once we export immutable stats.
+    def copy_view_data(self, view_data):
```

Comment:
nit: Not sure how others feel about naming, but maybe something like `finalize_view_data` to make it clear that this also modifies it? Not super important given that this should go away soon.

Reply:
Actually this method does both - it first makes a copy of the original `view_data` and then sets the `end_time` of that copy. I renamed it to `copy_and_finalize_view_data` to make this clear.